### ISSUE TITLE
show deleted files / trash size

### ISSUE THREAD

**Issue by wookietreiber** [nextcloud/server#14138]
**Is your feature request related to a problem? Please describe.**
The deleted files (view=trashbin) neither shows the size of the trash (the entire trash) nor the sizes of the files / directories contained in the trash. This makes it hard to decide when and what to remove if you don't simply want to empty the entire trash.

**Describe the solution you'd like**
Show the sizes like in the normal file view.

**Describe alternatives you've considered**
None.

**Additional context**

Trash shows no file sizes:

![screenshot-firefox-2019-02-11-09-19-34](https://user-images.githubusercontent.com/827088/52551793-424f6580-2dde-11e9-858b-90007e7226d2.png)

Normal folder view shows file sizes:

![screenshot-firefox-2019-02-11-09-22-08](https://user-images.githubusercontent.com/827088/52551883-94908680-2dde-11e9-911c-2c02321f61bf.png)

**Comment by alexanderdd** [comment]
Also, you should be able to **sort by file size**, as in the normal folder view (so you can start deleting the largest files).
Thanks to whoever is working on this.

**Comment by alexanderdd** [comment]
15.0.7 -- I would still like to see this enhancement, especially including sort by size. I need this feature a lot so I can free up disk space starting with the biggest files first.
(comment to remove stale label)

**Comment by alexanderdd** [comment]
It seems to me it should be fairly easy to implement the same file view for trash as in regular folder view, including file size and ability to sort by size, and deletion date. 

Or should I open a separate issue for "sorting files by size in trash view"?

**Comment by alexanderdd** [comment]
I can confirm that this is still not implemented in 19.0.4    =(

**Comment by alexanderdd** [comment]
Can you triage this please? @juliushaertl @skjnldsv

**Comment by juliushaertl** [comment]
Makes sense to have I'd say (cc @nextcloud/designers)

**Comment by nimisha-vijay** [comment]
Yep, all of that makes sense. The size in the trash bin can be exactly where it is for the normal files and it can allow sorting by size too.

**Comment by rakekniven** [comment]
Allow sorting the trash bin by size would be really helpful

**Comment by alexanderdd** [comment]
> Note: Note available at v24.0.2
did you mean "now available"? so this bug is closed?

**Comment by being-peace** [comment]
Related to https://github.com/nextcloud/server/issues/24544.

**Comment by szaimen** [comment]
Cc @skjnldsv the size is now shoen after the rewrite, right?

**Comment by skjnldsv** [comment]
Yep, done already! :rocket:
